src/main/java/core/machines/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.machines;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator stopped working");
```

Comment:
Similar to the `doWork` method, the message in `stopWork` should also be more informative and indicate the type of machine that stopped working. Update the message to reflect this, for example, 'Excavator has stopped its work.'